src/VisualStudio/Core/Def/Implementation/FindReferences/ToolTips/DisposableToolTip.cs

```diff
@@ -1,34 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System;
-using System.Diagnostics;
-using System.Windows.Controls;
-using Microsoft.CodeAnalysis.Editor.Shared.Preview;
-
-namespace Microsoft.VisualStudio.LanguageServices.FindUsages
-{
-    internal partial class StreamingFindUsagesPresenter
-    {
-        private class DisposableToolTip : IDisposable
-        {
-            public readonly ToolTip ToolTip;
-            private PreviewWorkspace _workspace;
-
-            private bool _disposed;
-
-            public DisposableToolTip(ToolTip toolTip, PreviewWorkspace workspace)
-            {
-                ToolTip = toolTip;
-                _workspace = workspace;
-            }
-
-            public void Dispose()
-            {
-                Debug.Assert(!_disposed);
-                _disposed = true;
-                _workspace.Dispose();
```

Comment:
that is not the problem. with this kind of tooltip, there is no way not holding roslyn solution since we show tooltip that shows a snapshot of world at the time FAR ran. without holding onto the solution, this kind of tooltip just fundamentally not possible unless we hold onto results of every IDE features such as classifications, squiggles, line separators, brace tagging, outlinging and etc.